src/pages/settings/company/documents/components/Table.tsx

```diff
@@ -37,11 +37,12 @@ export function Table() {
   const [t] = useTranslation();
   const [currentPage, setCurrentPage] = useState<number>(1);
   const [perPage, setPerPage] = useState<string>('10');
+  const companyDocuments = 'true';
```

Comment:
Does this need to be external variable? From what I can see, it's never changing?